pkg/tsdb/azuremonitor/azuremonitor-resource-handler.go

```diff
@@ -33,30 +33,31 @@ func (s *httpServiceProxy) writeErrorResponse(rw http.ResponseWriter, statusCode
 	rw.Header().Set("Content-Type", "application/json")
 	rw.WriteHeader(statusCode)
 
+	// Set error response to initial error message
+	errorBody := map[string]string{"error": message}
+
 	// Attempt to locate JSON portion in error message
 	re := regexp.MustCompile(`\{.*?\}`)
```

Comment:
How are error messages formatted? having a regex like this scares me when we could probably so a json.unmarshal